gadget-common/Cargo.toml

```diff
@@ -19,4 +19,8 @@ sc-client-api = { workspace = true }
 sp-core = { workspace = true }
 sp-runtime = { workspace = true }
 hex = { workspace = true }
-bincode2 = { workspace = true }
\ No newline at end of file
+bincode2 = { workspace = true }
+
+sp-api = { workspace = true, default-features = false, features = ["std"] }
+tangle-primitives = { workspace = true, default-features = false, features = ["std"] }
+pallet-jobs-rpc-runtime-api = { workspace = true, default-features = false, features = ["std"] }
```

Comment:
Lets make `std` feature in that crate, and only enable `std` for other packages/crates only if we enabled `std`